### ISSUE TITLE
Filter movies by genre (like in Web / Android / Desktop)

### ISSUE THREAD

**Issue by ngosang** [jellyfin/jellyfin-androidtv#4077]
### Describe the feature you'd like

In Jellyfin for  Web / Android / Desktop clients, when I click in a Library:
![image](https://github.com/user-attachments/assets/23f67537-ceca-407c-9e6e-738f9061db8f)

I see a filter for "Genre" among other things.
![image](https://github.com/user-attachments/assets/6f73b169-bcf2-42e3-abe8-6a6602cbf537)

Why is it not available in Jellyfin for Android TV? I'm doing something wrong?

**Comment by nielsvanvelzen** [comment]
Enable the "smart screen" using the gear icon in the library and you'll now have the option to browse by genre. This will improve in the future.

**Comment by ngosang** [comment]
Yeah, that option works for me but IMHO it's a bit hidden for the average user.
Futhermore, I don't know how to disable this option. I can't see the "gear" icon anymore.

**Comment by nielsvanvelzen** [comment]
When you use the "browse all" option you can find it again

**Comment by gizmomelb** [comment]
+1 requesting this feature, or at least making it more obvious and set by default please - especially for movies at least.

**Comment by 0ceanSlim** [comment]
This is my biggest complaint about the TV app. It's a huge miss imo. +1 for the request

**Comment by spilla7** [comment]
Another upvote for this issue here. This app is almost perfect (many thanks!!) but the ability to add genres to the smart screen or use the metadata genre filter like on the webui is very much needed. 

Alternatives like adding extra root folders is adding to much complication. Especially when using the "arr" apps.
